tests/fixtures/database.py

```diff
@@ -97,69 +99,31 @@ def _get_database_connection() -> Tuple[Engine, Connection]:
         return engine, connection
 
     @staticmethod
-    def create() -> "DatabaseFixture":
+    def _load_core_model_classes():
+        path = Path(__file__)  # tests/fixtures/database.py
+        path = path.parent  # tests/fixtures
+        path = path.parent  # tests
+        path = path.parent  # .
+        path = os.path.join(path, "core", "model")
+
+        list_modules = os.listdir(path)
+        list_modules.remove("__init__.py")
+
+        for module_file in list_modules:
```

Comment:
Do the modules need to be loaded on every request of the fixture or once during session? Since they're imported modules they probably stay imported after the first time

Reply:
This particular fixture has session scope, so they should only be loaded once:

```
@pytest.fixture(scope="session")
def database(application: ApplicationFixture) -> Iterable[DatabaseFixture]:
```